ts-pkgs/peer-e2e/tests/e2e/scripts/setup.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -meuo pipefail
+
+# Create a network for the guardians to comunicate
+docker network create dkg-test || true
```

Why is this ignoring error exit codes?